README.md

```diff
@@ -1,13 +1,39 @@
-# Terradactile
+<br />
+<p align="center">
+  <a href="https://github.com/sparkgeo/template-barebones">
+    <img src="./images/sparkgeo-logo.png" width="100">
+  </a>
+  <h3 align="center">Terradactile Lambda</h3>
+</p>
+
+
+<!-- TABLE OF CONTENTS -->
+### Table of Contents
+
+* [About this Program](#about-the-project)
+* [Getting Started](#getting-started)
+  * [Prerequisites](#prerequisites)
+  * [Installation and Deployment](#installation-and-deployment)
+* [Usage](#usage)
+* [Contact](#contact)
 
-Terradactile is an API and Lambda function for fetching, assembling, and processing elevation data.
 
-This project contains source code and supporting files for a serverless application that you can deploy with the SAM CLI. It includes the following files and folders.
+<!-- ABOUT THE PROJECT -->
+# About This Program
 
+Terradactile is an API and Lambda function for fetching, assembling, and processing elevation data.
+
+```markdown
+File Contents
 - terradactile/terradactile - Code for the application's Lambda function.
-- terradactile/template.yaml - A template that defines the application's AWS resources. **Update the environment variables for BUCKET and ALLOWED_ORIGINS.**
+- terradactile/template.yaml - A template that defines the application's AWS resources. 
 
-The application uses several AWS resources, including Lambda functions and an API Gateway API. These resources are defined in the `template.yaml` file in this project. You can update the template to add AWS resources through the same deployment process that updates your application code.
+**Update the environment variables for BUCKET and ALLOWED_ORIGINS.**
```

Comment:
Consider making this more explicit by adding a `.env-example` file with those in it. Or use a code block here such as:
```
export BUCKET=myBucket
export ALLOWED_ORIGINS=*
```